docs/conf.py

```diff
@@ -84,7 +85,23 @@
     "exclude-members": "__repr__,__weakref__,__metaclass__,__init__,model_config,model_fields,model_post_init"
 }
 
+# -- Doctest configuration -------------------------------------------------
+# Doctest global imports and instances
+import doctest
 
+doctest_default_flags = (
+    0
+    | doctest.DONT_ACCEPT_TRUE_FOR_1
+    | doctest.ELLIPSIS
+    | doctest.IGNORE_EXCEPTION_DETAIL
+    | doctest.NORMALIZE_WHITESPACE
+)
+
+doctest_global_setup = '''
+from ape import accounts
+
+my_account_alias = accounts.test_accounts[0]
```

Comment:
question: are we able to have individual test setups? something like this doesnt need to be global
issue: i commented elsewhere, but this is supposed to be an alias, not an account, and i think itll likely always fail, ideally we ignore it because it is like a "task failed successfully" kinda moment. but otherwise, you could try `my_account_alias = accounts.test_accounts[0].alias`

Reply:
Yes we can do individual test setups. I was trying this out to test if it would be nicer for this to have in a global setup but I'll switch it to individual since it's currently not being used for more than one test.